airbyte-cdk/python/airbyte_cdk/sources/declarative/retrievers/retriever.py

```diff
@@ -31,7 +32,7 @@ def read_records(
         """
 
     @abstractmethod
-    def stream_slices(self) -> Iterable[Optional[StreamSlice]]:
+    def stream_slices(self) -> Iterable[Optional[PerPartitionStreamSlice]]:
```

Comment:
I see the parent being updated but not the children (`SimpleRetriever` and `SimpleRetrieverTestReadDecorator`)